src/server/appserver.cpp

```diff
@@ -467,6 +469,57 @@ void AppServer::logExtendedLogActivationMessage(bool isExtendedLogEnabled) noexc
     LOG_INFO(_logger, msg.c_str());
 }
 
+void AppServer::setDefaultSentryUsers() {
+    std::vector<User> userList;
+    ParmsDb::instance()->selectAllUsers(userList);
+
+    std::string userId = "No user in db";
+    std::string userName = "No user in db";
+    std::string userEmail = "No user in db";
+    std::string allUsersIds = "No user in db";
+
+    if (!userList.empty()) {
+        userId = std::to_string(userList[0].userId());
+        userName = userList[0].name();
+        userEmail = userList[0].email();
+        for (const auto &user : userList) {
+            if (user.userId() != userList[0].userId()) {
```

Comment:
If the list can contain duplicate entries, shouldn't we use a set to remove duplicates?